### ISSUE TITLE
Formatting with long strings

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#124]
```
When you receive a tweet in yout timeline that contains a long word or any 
kind of string like "exampleofverylongstring0123456789" Pino should trunc it 
or something to put it in the size of tweetboxes you have... because right 
now it looks horrible... the layout expands its width, leading to the 
presence of an horizontal scrollbar.



```

Original issue reported on code.google.com by `mich.ba...@gmail.com` on 10 Mar 2010 at 5:29

Attachments:
- [Pantallazo.png](https://storage.googleapis.com/google-code-attachments/pino-twitter/issue-124/comment-0/Pantallazo.png)

**Comment by GoogleCodeExporter** [comment]
Original comment by `tro...@gmail.com` on 10 Mar 2010 at 9:21
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Closed by 7bd41a7c99 revision.
```

Original comment by `tro...@gmail.com` on 25 Mar 2010 at 12:19
- Changed state: **Fixed**
